content/api/index.html

```diff
@@ -137,6 +138,39 @@
       </div>
     </div>
 
+
+
+    <!--
+   =====================================================================
+      Rate Limiting
+    ====================================================================
+    -->
+
+    <div class="int-anchor" id="ratelimiting"></div><a href="#ratelimiting"><h3>Rate Limiting</h3></a>
+    <div class="row">
+      <%= left_side_div %>
+        <p>
+        Some of our API endpoints are rate limited.  Once you exceed a certain number of requests in a certain time period we return an error.
+        </p>
+        <p>
+        For rate limited API endpoints we return headers so you can know how close you are to your limit.  If you exceed your limit, you can review these headers to determine when you will be able to try again.
+        </p>
+        <p>
+        Rate limits can be increased from defaults by contacting the Datadog Support team.
```

Comment:
Maybe link "contacting the Datadog Support team" to http://docs.datadoghq.com/help/